src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/X86/Bmi2.PlatformNotSupported.cs

```diff
@@ -33,9 +33,18 @@ public abstract class X64
             /// <summary>
             /// unsigned __int64 _mulx_u64 (unsigned __int64 a, unsigned __int64 b, unsigned __int64* hi)
             ///   MULX r64a, r64b, reg/m64
```

Comment:
This could be better expressed as `MULX r64a, r64a, reg/m64` (both src1 and dest being the same register) as that is the actual instruction that will return only the high part (which should be called out here in docs).